forge-gui/res/cardsfolder/upcoming/fangorn_tree_shepherd.txt

```diff
@@ -0,0 +1,11 @@
+Name:Fangorn, Tree Shepherd
+ManaCost:4 G G G
+Types:Legendary Creature Treefolk
+PT:4/10
+S:Mode$ Continuous | Affected$ Treefolk.YouCtrl | AddKeyword$ Vigilance | Description$ Treefolk you control have vigilance.
+T:Mode$ AttackersDeclared | ValidAttackers$ Treefolk.YouCtrl | Execute$ TrigMana | TriggerZones$ Battlefield | TriggerDescription$ Whenever one or more Treefolk you control attack, add twice that much {G}.
+SVar:TrigMana:DB$ Mana | Produced$ G | Amount$ X
+S:Mode$ UnspentMana | ValidPlayer$ You | ManaType$ Green | Description$ You don't lose unspent green mana as steps and phases end.
+SVar:X:Count$Valid Treefolk.YouCtrl+attacking/Times.2
```

Comment:
Better use TriggeringObject because the amount shouldn't change if one gets removed in response

Reply:
Hmm orginal Grand Warlord Radha uses same template as this. 

Would this work SVar:X:TriggeredObjects$Valid Treefolk.YouCtrl+attacking/Times.2

Reply:
Then please fix both:
> The amount of mana you’ll add is the number of creatures you attack with. Creatures that are put onto the battlefield attacking before Radha’s triggered ability resolves don’t count, and **creatures that attacked but left combat before the triggered ability resolves do count**. 

I meant just `TriggeredAttackers`

Reply:
`TriggeredAttackers$Amount`

Reply:
SVar:X:TriggeredAttackers$Amount causes crash :(

Reply:
Thanks for the help